README.fr.md

```diff
@@ -2808,6 +2809,41 @@ Avec les options :
 -x # active le mutex lock (off par défaut)
 ```
 
+### Multi-threading
+
+_Cette partie est basée sur [une réponse StackOverflow][so-answer] par Konstantin._
+
+Sinatra n'impose pas de modèle de la concurrence. Sinatra est thread-safe, vous pouvez
+donc utiliser n'importe quel gestionnaire Rack, comme Thin, Puma ou WEBrick en mode
+multi-threaded.
+
+Lorsque le serveur est démarré, vous devez spécifier l'appel de méthode correcte pour
```

Comment:
What do you guys think about:

> Afin de démarrer le serveur, il vous faut spécifier la commande associée à votre gestionnaire Rack.

At least, "when starting the server" can't be translated with "Lorsque le serveur est démarré" because the latter makes the assumption that the server is already started while we should specify a command to do so.